akka-docs/src/main/paradox/typed/cluster.md

```diff
@@ -95,15 +98,327 @@ Scala
+
+## Serialization 
+ 
+Enable @ref:[serialization](../serialization.md) to send events between ActorSystems and systems
+external to the Cluster. @ref:[Serialization with Jackson](../serialization-jackson.md) is a good choice in many cases, and our
+recommendation if you don't have other preferences or constraints.
+ 
+Actor references are typically included in the messages, since there is no `sender`. 
+To serialize actor references to/from string representation you would use the `ActorRefResolver`.
+For example here's how a serializer could look for `Ping` and `Pong` messages:
 
 Scala
 :  @@snip [PingSerializer.scala](/akka-cluster-typed/src/test/scala/docs/akka/cluster/typed/PingSerializer.scala) { #serializer }
 
 Java
 :  @@snip [PingSerializerExampleTest.java](/akka-cluster-typed/src/test/java/jdocs/akka/cluster/typed/PingSerializerExampleTest.java) { #serializer }
+
+You can look at the
+@java[@extref[Cluster example project](samples:akka-samples-cluster-java)]
+@scala[@extref[Cluster example project](samples:akka-samples-cluster-scala)]
+to see what this looks like in practice.
+
+## Node Roles
+
+Not all nodes of a cluster need to perform the same function: there might be one sub-set which runs the web front-end,
+one which runs the data access layer and one for the number-crunching. Deployment of actors—for example by cluster-aware
+routers—can take node roles into account to achieve this distribution of responsibilities.
+
+The node roles are defined in the configuration property named `akka.cluster.roles`
+and typically defined in the start script as a system property or environment variable.
+
+The roles are part of the membership information in `MemberEvent` that you can subscribe to.
+
+## Failure Detector
+
+The nodes in the cluster monitor each other by sending heartbeats to detect if a node is
+unreachable from the rest of the cluster and has been downed and removed. Please see:
+
+* @ref:[Failure Detector specification](cluster-specification.md#failure-detector)
+* @ref:[Phi Accrual Failure Detector](failure-detector.md) implementation
+* [Using the Failure Detector](#using-the-failure-detector) 
+ 
+### Using the Failure Detector
+ 
+Cluster uses the `akka.remote.PhiAccrualFailureDetector` failure detector by default, or you can provide your by
+implementing the `akka.remote.FailureDetector` and configuring it:
+
+```
+akka.cluster.implementation-class = "com.example.CustomFailureDetector"
+```
+
+In the [Cluster Configuration](#configuration) you may want to adjust these
+depending on you environment:
+
+* When a *phi* value is considered to be a failure `akka.cluster.failure-detector.threshold`
+* Margin of error for sudden abnormalities `akka.cluster.failure-detector.acceptable-heartbeat-pause`  
+
+If you encounter suspicious false positives when the system is under load you should
+define a separate dispatcher for the cluster actors as described in [Cluster Dispatcher](#cluster-dispatcher).
```

Comment:
They are already isolated (2.6) on the dedicated internal Akka dispatcher for internal actors (already mentioned below)

Reply:
Thats from right above this header, https://doc.akka.io/docs/akka/current/cluster-usage.html#how-to-test
```
If you encounter suspicious false positives when the system is under load you should define a separate dispatcher for the cluster actors as described in Cluster Dispatcher.
```

Reply:
Shall we remove that statement then?

Reply:
I think so, it was probably missed when I moved everything to the isolated dispatcher earlier this summer.

Reply:
Done :)